ZUGFeRD/IncludedItem.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace s2industries.ZUGFeRD
+{
+    /// <summary>
+    /// An included Item referenced from this trade product.
+    /// </summary>
+    public class IncludedItem
```

Comment:
It could be I am missing something (I am not used to this kind of documentation) but I think we're not talking about the same thing. I used the "3. FACTUR-X 1.0.07 FR-EN.xlsx"-File to implement mandatory fields of BG-X-1. Now that I looked at the "8. Factur-X_1.0.07_Technischer-Anhang_Profil_EXTENDED.pdf"-File, maybe the class name "IncludedItems" is a bit misleading and I should have used "IncludedReferencedProduct" instead. 
![grafik](https://github.com/user-attachments/assets/7f7b6bd6-baf5-4e68-898c-4a06f3f36441)